:memo: Update broken transifex links

homersimpsons opened the pull request:
Updates all `www.transifex.com` links to their `app.transifex.com` counterpart (will redirect to  `explore.transifex.com` if the user is not logged in) and also `docs.transifex.com` to `help.transifex.com`.

homersimpsons committed:
:memo: Update broken transifex links

homersimpsons commented on RELEASING.md:
```diff
@@ -1,7 +1,7 @@
 
 ### Prerelease (several days prior)
-- [Notify translators](https://www.transifex.com/projects/p/id-editor/announcements/) of the impending release
+- [Notify translators](https://app.transifex.com/openstreetmap/communication/?q=project%3Aid-editor) of the impending release
```
I can access this URL but I do not know if an announcement can be created from here. I read the steps on https://help.transifex.com/en/articles/6224264-announcements-discussions-messages so the URL should be okay. Someone with access should test this

Note that I tested https://app.transifex.com/openstreetmap/id-editor/communication but this does not work

tyrasd replied:
The new link is perfect: as an admin one can create new announcements from there. Thanks :+1:

homersimpsons commented on CONTRIBUTING.md:
```diff
@@ -211,9 +211,9 @@ switch to the development database.
 
 Translations are managed using the
-[Transifex](https://www.transifex.com/openstreetmap/id-editor/) platform. After
+[Transifex](https://explore.transifex.com/openstreetmap/id-editor/) platform. After
```
This first link redirected to the project and not to transifex itself, I believe this was intentional.

homersimpsons commented on ACCESSIBILITY.md:
```diff
@@ -169,7 +169,7 @@ for more info.
 | ❌ | Language selection in UI | The mapper should be able to view and change iD's language in the interface at any time. Useful for public computers with fixed browser languages | [#3120](https://github.com/openstreetmap/iD/issues/3120) |
 | 🟩 | Right-to-left layouts | The [`dir` HTML attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/dir) is properly set for languages like Hebrew and Arabic |
-| ✅ | [Language-specific plurals](https://docs.transifex.com/localization-tips-workflows/plurals-and-genders#how-pluralized-strings-are-handled-by-transifex) | English has two plural forms, but some languages need more to be grammatically correct | [#597](https://github.com/openstreetmap/iD/issues/597), [#7991](https://github.com/openstreetmap/iD/issues/7991) |
+| ✅ | [Language-specific plurals](https://help.transifex.com/en/articles/6231958-working-with-plurals-and-genders#h_51e09ec18a) | English has two plural forms, but some languages need more to be grammatically correct | [#597](https://github.com/openstreetmap/iD/issues/597), [#7991](https://github.com/openstreetmap/iD/issues/7991) |
```
I do not know how stable is this anchor but I believe that it makes more sense to have it correct at least for now.

homersimpsons commented:
By the way I wanted to update the french transhlation for https://app.transifex.com/openstreetmap/id-editor/translate/#fr/presets/41336797?q=key%3Aincline+reviewed%3Ano but it was updated correctly 8 years ago (on april 2017). ~~Could the translations here be refreshed? Should I submit a pull request for this?~~ So it seems there is an issue somewhere in the process, because I can still see the old translation in https://www.npmjs.com/package/@openstreetmap/id-tagging-schema?activeTab=code in file `dist/translations/fr-FR.json:2129`:
```
                "incline_steps": {
                    "options": {
                        "down": "Vers le bas",
                        "up": "En haut" // <- this is wrong
                    }
                },
```

~~Looking at the code I could find https://github.com/openstreetmap/id-tagging-schema/blob/c3e2fad6f26738bec779269c818eea1877fec5aa/data/fields/incline_steps.json but couldn't find any place with the translations.~~

tyrasd commented:
> `incline_steps`

well, in the `fr_FR` locale (_French (France)_), it was fixed a few weeks ago (to the text: _“Vers le haut”_). It will be included in the upcoming release of the tagging schema in the coming days.

That said, does it really make sense to even have separate translations for "French in France" for iD on transifex? :thinking:

homersimpsons commented:
> > `incline_steps`
> 
> well, in the `fr_FR` locale (_French (France)_), it was fixed a few weeks ago (to the text: _“Vers le haut”_). It will be included in the upcoming release of the tagging schema in the coming days.
> 
> That said, does it really make sense to even have separate translations for "French in France" for iD on transifex? 🤔

Oh I never noticed that! i fixed a translation a month ago and wasn't seeing it but this explains everything! I just fixed it again in `fr_FR` so I should see the change on the next release.

I do not know if both makes sense. i'm wondering if there is a concept such as "extension", meaning that `fr_FR` should derive from `fr` and inherit mostly everything, but being able to override some. Eit: looks like there is a similar concept behind https://help.transifex.com/en/articles/6224636-introduction-to-translation-memory

tordans commented on ACCESSIBILITY.md:
```diff
@@ -193,12 +193,12 @@ translated to one or more languages.
 
 The completion percentages for iD translations constantly change, and so are not
-listed here. Visit the [Transifex project page](https://www.transifex.com/openstreetmap/id-editor/)
+listed here. Visit the [Transifex project page](https://explore.transifex.com/openstreetmap/id-editor/)
```
```suggestion
listed here. Visit the [Transifex project page](https://app.transifex.com/openstreetmap/id-editor/)
```

When logged in, app. is better. When logged out, it redirects to https://explore.transifex.com/openstreetmap/id-editor/ …

homersimpsons replied:
Done, amend

tordans commented on ACCESSIBILITY.md:
```diff
@@ -193,12 +193,12 @@ translated to one or more languages.
 | ✅ | iD validation issues | | |
 | ✅ | KeepRight issues | | |
-| ✅ | Osmose issues | Translated strings are [provided by Osmose](https://www.transifex.com/openstreetmap-france/osmose/) itself, not iD | |
+| ✅ | Osmose issues | Translated strings are [provided by Osmose](https://explore.transifex.com/openstreetmap-france/osmose/) itself, not iD | |
```
```suggestion
| ✅ | Osmose issues | Translated strings are [provided by Osmose](https://app.transifex.com/openstreetmap-france/osmose/) itself, not iD | |
```

See below

homersimpsons replied:
Done, amend

tordans commented on CHANGELOG.md:
```diff
@@ -6432,7 +6432,7 @@ _Check out the new imagery by opening the Background pane (shortcut <kbd>B</kbd>
 
 * Update to Mapillary API v3, use traffic signs from Mapillary sprites ([#4050], thanks [@nickplesha])
-* iD editor translation project on Transifex has moved to the [OpenStreetMap organization](https://www.transifex.com/openstreetmap/)
+* iD editor translation project on Transifex has moved to the [OpenStreetMap organization](https://explore.transifex.com/openstreetmap/)
```
```suggestion
* iD editor translation project on Transifex has moved to the [OpenStreetMap organization](https://app.transifex.com/openstreetmap/)
```

homersimpsons replied:
Done, amend

tordans commented on CONTRIBUTING.md:
```diff
@@ -211,9 +211,9 @@ switch to the development database.
 
 Translations are managed using the
-[Transifex](https://www.transifex.com/openstreetmap/id-editor/) platform. After
+[Transifex](https://explore.transifex.com/openstreetmap/id-editor/) platform. After
```
```suggestion
[Transifex](https://app.transifex.com/openstreetmap/id-editor/) platform. After
```

homersimpsons replied:
Done, amend

tordans commented on CONTRIBUTING.md:
```diff
@@ -211,9 +211,9 @@ switch to the development database.
 signing up, you can go to [iD's project
-page](https://www.transifex.com/openstreetmap/id-editor/), select a language and
+page](https://explore.transifex.com/openstreetmap/id-editor/), select a language and
 click **Translate** to start translating. Translations are divided into
```
```suggestion
page](https://app.transifex.com/openstreetmap/id-editor/) and
click **Translate** to start translating. Translations are divided into
```

I don't have to "select a language", the "Translate" Button is right on the page. Right?

homersimpsons replied:
Done, amend

tordans replied:
@homersimpsons the removal of "select a language" did not stick; i will apply it manually right after posting this. This is just FYI just in case I am missing something.

tordans commented on CONTRIBUTING.md:
```diff
@@ -235,11 +235,11 @@ For more information on translating the presets [please see this id-tagging-sche
 which is updated every time we change the `develop` branch.
 
-[iD translation project on Transifex](https://www.transifex.com/openstreetmap/id-editor/)
+[iD translation project on Transifex](https://explore.transifex.com/openstreetmap/id-editor/)
```
```suggestion
[iD translation project on Transifex](https://app.transifex.com/openstreetmap/id-editor/)
```

homersimpsons replied:
Done, amend

tordans reviewed (changes requested):
In general all changes do work. I suggest to change them to the app.* domain instead of explore.* because App will redirect to explore if logged out but when logged in it's the better place to be.

We should also double check if the instructions are still correct. I suggested one removal of "select the translation" which is how the flow is today IMO.

homersimpsons commented:
I updated the `explore.transifex.com` links to `app.transifex.com`. Note that there was a link in `CONTRIBUTING.md` that was not part of the previous diff that I also updated to stay consistent:
```diff
-[Osmose Transifex project](https://explore.transifex.com/openstreetmap-france/osmose/)
+[Osmose Transifex project](https://app.transifex.com/openstreetmap-france/osmose/)
```

tordans committed:
Remove outdated "select a language" from docs
